tests/Maker/FunctionalTest.php

```diff
@@ -162,10 +162,10 @@ public function getCommandTests()
         yield 'fixtures' => [MakerTestDetails::createTest(
             $this->getMakerInstance(MakeFixtures::class),
             [
-                'AppFixtures',
+                'AcmeFixtures',
             ])
             ->assert(function (string $output, string $directory) {
-                $this->assertContains('created: src/DataFixtures/AppFixtures.php', $output);
+                $this->assertContains('created: src/DataFixtures/AcmeFixtures.php', $output);
```

Comment:
Don't know if we need here to check it. I think we can remove assert block here, tests automatically check if command successfully executed.

Reply:
The assertion indicates whether the correct file is created, even though it just relies on the message.
Anyway, the PR is just for fixing the error in the test.